### DISCUSSION TITLE
Mastodon won't allow me to access after switching to a new computer

### DISCUSSION THREAD

**Discussion by bh-siteobjective** [mastodon/mastodon#21824] [General]
Hi there, 

This all started when switching to a new Mac. I tried to access my Mastodon account on seocommunity.social via Google Chrome - the same Chrome account that I used to access it on my former computer. But, I have been unable to access Mastodon (despite starting on the server I enrolled on) and I only get a blank page. This is despite flushing my system cache and the page level cache.

Best regards, 

Brian

**Comment by nightpool** [comment]
@bh-siteobjective This sounds like a problem with your Chrome version. When you say you "Only get a blank page", what does this mean? Is there anything in the console? I would suggest talking to your admin, they'll be able to provide you much better support then we can
